regparser/grammar/terms.py

```diff
@@ -16,6 +16,7 @@
 
 e_tag = (
     Suppress(Regex(r"<E[^>]*>"))
+    + NotAny(Marker("this") + Marker("term")) 
```

Comment:
I'm not sure I follow -- are you saying that there's text like ```<E>This term</E> can also mean "a delicate cheese"```? If so, I'm not sure that the grammar is the best place to exclude those "definitions". There's some existing logic (perhaps configurations) around excluding phrases/false positives in the `Terms` layer. Would that be more appropriate place for this?